apps/atqr/web/src/pages/timeForTheTruth/timeForThruth.pug

```diff
@@ -0,0 +1,38 @@
+doctype html
```

Comment:
Your pug file should be running.  The cdns and ts file imports are missing. Do this and run with parcel